public/lessons/www/html.md

```diff
@@ -1,25 +1,26 @@
-# HTML
+# Intro to HTML
 
-Hyper-Text Markup Language
-
-one HTML file = one page
-
-# History
+HTML stands for Hyper-Text Markup Language. Every HTML file = one web page. It is the standard language used for creating web pages. 
```

Comment:
Yes. I tend to be writing more, and using fewer bullet points - I think especially since these are slides, bullet points are better. I'll make this change.